documentation/modules/cruise-control/con-optimization-proposals.adoc

```diff
@@ -107,16 +177,17 @@ m¦excludedBrokersForReplicaMove
 
 
 [discrete]
-=== Broker load
+=== Broker load properties
 
 The broker load is stored in a ConfigMap (with the same name as the KafkaRebalance custom resource) as a JSON formatted string. This JSON string consists of a JSON object with keys for each broker IDs linking to a number of metrics for each broker.
 Each metric consist of three values.
 The first is the metric value before the optimization proposal is applied, the second is the expected value of the metric after the proposal is applied, and the third is the difference between the first two values (after minus before).
 
 NOTE: The ConfigMap appears when the KafkaRebalance resource is in the `ProposalReady` state and remains after the rebalance is complete.
 
-To extract the JSON string from the ConfigMap you could use the following command, which uses the jq command line JSON parser tool:
+You can use the `jq` command line JSON parser tool to extract the JSON string from the ConfigMap.
```

Comment:
What if `jq` is not installed and user needs to extract the JSON string? Should we give some alternative for that?

Reply:
Thanks @ShubhamRwt - I guess we can add something like the following:

> You can use the name of the ConfigMap to view its data from the command line.
> 
> **Returning ConfigMap data**
> `kubectl describe configmaps _<my_rebalance_configmap_name>_ -n _<namespace>_`
> 
> You can also use the `jq` command line JSON parser tool to extract the JSON string from the ConfigMap....